exoctk/exoctk_app/log_exoctk.py

```diff
@@ -34,12 +34,15 @@ def create_db(dbpath, overwrite=True):
         if not dbpath.endswith('.db'):
             raise IOError('Please provide a path with a .db file extension')
 
+        # Use pathlib.Path to become compliant with bandit.
+        p = Path(dbpath)
+        
         # Remove existing file if overwriting
         if os.path.isfile(dbpath) and overwrite:
-            os.system('rm {}'.format(dbpath))
+            p.unlink()
 
         # Make the new file
-        os.system('touch {}'.format(dbpath))
+        p.touch()
```

Comment:
@bourque @mfixstsci 
I like this way of file I/O -- definitely better than `os.system`. I believe when I had looked into this in the past, I had seen that the "pythonic" recommendation was `shutil`. Has `Path` surpassed this and/or is `shutil` lacking functionality?

Reply:
I don't know if it has surpassed it but I know that this a more secure way of handling the creation and deletion of paths, files etc. I am not sure if `shutil` is or isn't compliant with `bandit`. I believe that `Path` was suggested by Jeff when we met with the exomast folks.

Reply:
@mfixstsci Did some reading -- looks like `pathlib` is being called the summation of `os.path`, `shutil`, and `glob`, so I'm comfy moving forward.